CardManagementSDK/Source/Network/ResponseModels/ViewPinResponse.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  ViewPinResponse.swift
+//  NICardManagementSDK
+//
+//  Created by Gabriel Cernestean on 29.08.2023.
+//
+
+import Foundation
+
+class ViewPinResponse: NSObject, Codable {
+    
+    var encryptedPin: String?                   /// Encrypted PIN under the provided public key
+ 
+    enum CodingKeys: String, CodingKey {
+        case encryptedPin = "encrypted_pin"
+    }
+    
+    func parse(json: Data) -> ViewPinResponse? {
+        let decoder = JSONDecoder()
+        do {
+            let response = try decoder.decode(ViewPinResponse.self, from: json)
+            return response
+        } catch DecodingError.dataCorrupted(let context) {
+            print(context)
+        } catch DecodingError.keyNotFound(let key, let context) {
+            print("Key '\(key)' not found:", context.debugDescription)
+            print("codingPath:", context.codingPath)
+        } catch DecodingError.valueNotFound(let value, let context) {
+            print("Value '\(value)' not found:", context.debugDescription)
+            print("codingPath:", context.codingPath)
+        } catch DecodingError.typeMismatch(let type, let context) {
+            print("Type '\(type)' mismatch:", context.debugDescription)
+            print("codingPath:", context.codingPath)
+        } catch {
+//            print("error: ", error)
```

Comment:
Why?

Reply:
<img width="337" alt="Screenshot 2023-11-01 at 14 29 11" src="https://github.com/network-international/card-management-sdk-ios/assets/72929424/22593a53-45a3-4635-b57d-1c5cb224e3c3">
<img width="338" alt="Screenshot 2023-11-01 at 14 29 24" src="https://github.com/network-international/card-management-sdk-ios/assets/72929424/d3853e7e-7c97-405f-945a-4227e6852d62">
Did some tests and it looks good with phone in arabic (RTL)

Reply:
Fixed.